src/main/java/org/skriptlang/skript/lang/experiment/Feature.java

```diff
@@ -0,0 +1,83 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.lang.experiment;
+
+import ch.njol.skript.SkriptAddon;
+import ch.njol.skript.patterns.PatternCompiler;
+import ch.njol.skript.patterns.SkriptPattern;
+
+/**
+ * Experimental feature toggles as provided by Skript itself.
+ */
+public enum Feature implements Experiment {
+	;
```

Comment:
whats this little guy up to

Reply:
Correct me if I'm wrong, but I believe he's needed if there's anything (fields/constructur/method) after enums.
He may seems lonely as he doesn't have any enum friends to keep him company.
If I were to venture a guess, Id assume Kenzie did this as Skript doesn't have any features yet.

Reply:
> whats this little guy up to

He's keeping the seat warm!
(Enum classes need a `;` before you can write any non-enum stuff!)